recipe/meta.yaml

```diff
@@ -44,7 +44,7 @@ requirements:
     - r-knitr
     - r-magrittr
     - imagemagick
-    - librsvg
+    - librsvg 2.44.*
```

Comment:
Narf, do you know why that is? It seems to be in the global pinnings :( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L414

Reply:
It's getting ignored and 2.40.* is getting installed. I don't know why that's happening, but that's the problem.

Reply:
Ah

    conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
      - librsvg=2.44
      - r-base=3.5.1

Reply:
Aha, r-base needs to be rebuilt with the current cairo pinning!

Reply:
That makes more sense, but more complicated and r-base will probably fail :(

Reply:
This was when it changed: https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/665c45c430fd214466c51049f855113d9e91737b

Reply:
I'll make a PR and we'll see how it goes.